plugins/woocommerce/tests/e2e-pw/tests/merchant/create-variable-product.spec.js

```diff
@@ -82,18 +73,13 @@ test.describe.serial( 'Add New Variable Product Page', () => {
 		await expect(
 			page.getByText( 'Product draft updated. ' )
 		).toBeVisible();
-		await page.click( '.updated.notice .notice-dismiss' );
 
 		// manually create variations from all attributes
 		await page.click( 'a[href="#variable_product_options"]' );
-		await page.selectOption( '#field_to_edit', 'link_all_variations', {
-			force: true,
-		} );
+		await page.selectOption( '#field_to_edit', 'link_all_variations' );
 		page.on( 'dialog', ( dialog ) => dialog.accept() );
-		await page.click( 'a.do_variation_action' );
-
-		await page.waitForLoadState( 'networkidle' );
 
+		await page.click( 'a.do_variation_action' );
 		// add variation attributes
 		for ( let i = 0; i < 8; i++ ) {
```

Comment:
When the test is working properly, there are actually 8 variations created.  So update this number (back) to what it was.

Reply:
All 24 assertions also working well when I looked at the trace. Looking good!